wip_pacta_data_qa.Rmd

```diff
@@ -0,0 +1,545 @@
+  filter((flag == "Missing currency information") &
+           investor_name != "Meta Investor") %>% 
+  group_by(investor_name, portfolio_name, currency, isin, exchange_rate_usd, company_name) %>% 
+  summarise(market_value_miss_currency = round(sum(market_value, na.rm = T))) %>% 
+  ungroup() %>% 
+  left_join(
+    investors_portfolio_overview %>% select(investor_name, portfolio_name, sum_market_value), by = c("investor_name", "portfolio_name")
+    ) %>% 
+  mutate(share_missing_market_value = round(
+    market_value_miss_currency/sum_market_value, 4))
+datatable(missing_currency, rownames = F)
+
+```
+
+## Missing Bloomberg data
+
+### Overview of holdings with missing Bloomberg data
+Check if data for these ISINs can be downloaded from Bloomberg and added to DataStore by Taylor
+Use the .csv file generated by the PACTA_analysis script to pass on the information. This DataTable is supposed to help understand more about individual cases.
+For ISINs that are not in DataStore, check if these are unlisted items (??)
+
+* **share_missing_value** shows the overall impact per ISIN and asset_type
+
+**OPEN:** should financial_sector == 'Other' be filtered out for convenience?
+
+```{r}
+not_in_bloomberg <- result_data_qa %>% 
+  filter(flag == "Holding not in Bloomberg database" &
+           investor_name != "Meta Investor") %>% 
+  select(isin, investor_name, portfolio_name, #market_value, currency, exchange_rate,
+         value_usd, asset_type, financial_sector, direct_holding) %>%
+  group_by(isin) %>% 
+  mutate(sum_value_usd = round(sum(value_usd, na.rm=T))) %>% 
+  ungroup() %>% 
+  left_join(total_overview, by = c("asset_type")) %>% 
+  mutate(share_missing_value = round(sum_value_usd.x/sum_value_usd.y, 5)) %>% 
+  select(-c(total_holdings)) %>% 
+  arrange(asset_type, isin)
+datatable(not_in_bloomberg)
+
+```
+
+
+## Missing or invalid ISIN
+
+### overview invalid or missing ISIN
+Overview of holdings with flag "Invalid or missing ISIN" or that have an ISIN with "MissingValue"
+In order to provide more insight as to what the reasons may be, additional info, such as fund_isin, currency and the flag are provided.
+
+* **share_missing_value** is the value in USD aggregated by fund_isin, relative to the analysis' overall value in USD
+* all holdings that do not have an entry for the fund_isin will accordingly be grouped as one set of holdings
+
+**Actions:**
+
+* if the ISIN is a "MissingValue" because it is not available in the BBG database, there is nothing we can do (??)
+* if the ISIN seems to have an odd format, check (potentially with the project partner) if the identifier is really aan ISIN, or some other identifier such as a FIGI. If so, update the identifier.
+* the invalid ISIN might also just be a typo, verify this for cases that look close to valid.
+* These actions are taken on the portfolio input level, so no need to involve the data team
+
+```{r}
+ISIN_missing_invalid <- result_data_qa %>% 
```

Comment:
the share_missing_value values here seem to be all the same despite differences in the value_usd. Not sure why

Reply:
looks like I grouped by fund_isin (which is NA for all these) instead of by isin. Not sure why I did that. Will check with Laura on Monday if this was how we did that and if so why. But for the time being, will change to isin level. Seems more reasonable indeed.